hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -91,7 +92,7 @@
  */
 public class ThrottleAccumulator {
     private static final Logger log = LogManager.getLogger(ThrottleAccumulator.class);
-    private static final Set<HederaFunctionality> GAS_THROTTLED_FUNCTIONS =
+    private static final Set<HederaFunctionality> CONTRACT_FUNCTIONS =
```

Comment:
Nit: `GAS_THROTTLED_FUNCTIONS` seems more appropriate since the set still includes `ETHEREUM_TRANSACTION`. Subjective opinion though